lib/engine/game/g_1880/step/check_fi_connection.rb

```diff
@@ -87,12 +87,26 @@ def process_percent(action)
 
           def process_token(action)
             fi = @merging[:fi]
+<<<<<<< HEAD
```

Comment:
This merge conflict needs to be resolved.

Reply:
i removed it....where are you seeing this?

Reply:
My bad, I'd clicked onto the wrong page and was looking at the first commit, not the full pull request.